packages/default-theme/plugins/api-client.js

```diff
@@ -0,0 +1,24 @@
+import { setup, onConfigChange } from '@shopware-pwa/shopware-6-client'
+
+export default ({ app }) => {
+  if (!app.$cookies) {
+    throw 'Error cookie-universal-nuxt module is not applied in nuxt.config.js'
+  }
+  const contextToken = app.$cookies.get('sw-context-token') || ''
+  /**
+   * Setup Shopware API client
+   */
+  setup({
+    endpoint: 'https://shopware-2.vuestorefront.io/sales-channel-api/v1',
+    accessToken: 'SWSCBVBBZET1RTFIYWY4YVLICA',
```

Comment:
```suggestion
    accessToken: 'SWSCMUDKAKHSRXPJEHNOSNHYAG',
```

now we should be using new accessKey related to the storefront channel in order to get the page resolver working well.